website/content/docs/enterprise/pkcs11-provider/aws-xks.mdx

```diff
@@ -0,0 +1,201 @@
+---
+layout: docs
+page_title: AWS KMS External Key Store (XKS) - PKCS#11 Provider - Vault Enterprise
+description: |-
+  The Vault PKCS#11 Provider can be used with AWS KMS External Key Store.
```

Comment:
nit: It probably makes more sense to readers if we start with a more zoomed out view:

```suggestion
  AWS KMS External Key Store can use Vault as a key store via the Vault PKCS#11 Provider.
```